packages/components/src/components/hds/app-side-nav/index.ts

```diff
@@ -19,32 +17,16 @@ interface HdsAppSideNavSignature {
     onDesktopViewportChange?: (arg: boolean) => void;
   };
   Blocks: {
-    header?: [
-      {
-        Header?: HdsAppSideNavBaseSignature['Blocks']['header'];
-        isMinimized?: boolean;
-      },
-    ];
-    body?: [
-      {
-        Body?: HdsAppSideNavBaseSignature['Blocks']['body'];
-        isMinimized?: boolean;
-      },
-    ];
-    footer?: [
-      {
-        Footer?: HdsAppSideNavBaseSignature['Blocks']['footer'];
-        isMinimized?: boolean;
-      },
-    ];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    default?: any;
```

Comment:
[question] @alex-ju I see different approaches here: in some cases `default: [];`, in others `default?: [];` and here `default?: any;`. Is this something that is worth to "standardize"?

Reply:
I would err towards anything that doesn't require disabling linting/`any` at least.

Reply:
I've been having issues in some cases using `default: [];`. I'll check this one again though.

Reply:
Just tried changing it back to using `default: []` and it's not complaining now as it was before so must have been related something else that I have since updated.

Pushed up.

Reply:
@didoo `default: [];` and `default?: [];` depend on wether it's optional or not; `default?: any;` is legacy declaration – as discussed during the engineering sync, it's on my list to align them but haven't got the time yet 😞

Reply:
ah, we only have one `any` in Blocks in SideNav that's deprecated, so not sure of the value at this point, but will try to follow up